.vscode/settings.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
Remove this file. Use this command to remove it from the repository, but keep it locally:

```
git rm --cached .vscode/settings.json
```

Then add it to the .gitignore file so it doesn't get committed again in the future.